CHANGELOG.md

```diff
@@ -81,6 +81,8 @@ Main (unreleased)
 
 - Allow setting the node name for clustering with a command-line flag. (@tpaschalis)
 
+- Set securityContext for configReloader container. (@yanehi)
```

Comment:
The Helm chart has its own CHANGELOG.md in the operations/helm directory, can you move this there instead?

Reply:
Done!